conf/app.routes

```diff
@@ -4,12 +4,11 @@ GET         /assets/*file                        controllers.Assets.at(path="/pu
 
 GET         /unauthorized                        @uk.gov.hmrc.tariffclassificationfrontend.controllers.SecurityController.unauthorized()
 
-GET         /                                    controllers.Default.redirect(to = "/tariff-classification/queues/my-cases")
-GET         /queues                              controllers.Default.redirect(to = "/tariff-classification/queues/my-cases")
-GET         /queues/my-cases                     @uk.gov.hmrc.tariffclassificationfrontend.controllers.MyCasesController.myCases()
+GET         /                                    @uk.gov.hmrc.tariffclassificationfrontend.controllers.MyCasesController.myCases()
+GET         /queues                              controllers.Default.redirect(to = "/tariff-classification")
 GET         /queues/:slug                        @uk.gov.hmrc.tariffclassificationfrontend.controllers.QueuesController.queue(slug: String)
 
-GET         /cases                               controllers.Default.redirect(to = "/tariff-classification/queues/my-cases")
```

Comment:
Double-check these changes with @ed0906 

I think Ed made some changes on the routes and default page.
I am worried you are reverting his changes...

Reply:
I’ve copied app.routes from master and added my changes back in. Should look OK now